metadata/demographic/census_place_points_2010.md

```diff
@@ -0,0 +1,74 @@
+# Title
+
+Utah Census Place Points 2010
+
+## ID
+
+ed312c4c-c0b7-41dc-b77e-50649bed8ebd
+
+## Brief Summary
+
+Points dataset of census places with age, race, income, and other demographic data.
+
+## Summary
+
+This dataset contains point locations for cities, downs, and Census Designated Places in Utah as of the 2010 Census. This dataset also includes demographic data that was current as of 2011. For 2020 Census data products, please see the [Demographic Data Index](https://gis.utah.gov/products/sgid/demographic/) available from the SGID.
+
+## Description
+
+### What is the dataset?
+
+Census Places include cities, towns, and Census Designated Places (CDP). The [US Census Bureau](https://www.census.gov/programs-surveys/bas/information/cdp.html) defines CDPs as "statistical geography representing closely settled, unincorporated communities that are locally recognized and identified by name." Both these unincorporated communities and legally incorporated municipalities are represented in this dataset.
+
+### What is the purpose of the dataset?
+
+The US Census Bureau uses both legally incorporated places (cities, towns, etc) and unincorporated places (CDPs) for the Census. You can read more about how GIS is used in other areas of the Census on the Kem C. Gardner Policy Institute's 2020 Census Geography [Blog](https://gardner.utah.edu/blog/blog-whats-new-in-utahs-census-2020-geography/) and [fact sheet](https://d36oiwf74r1rap.cloudfront.net/wp-content/uploads/Geog-FS-Mar2021.pdf).
+
+### What does the dataset represent?
+
+Each point in this dataset was placed in the geographic center of a city, town, or CDP boundary. Features in this dataset contain a wide variety of demographic data, including income, age, race, and ethnicity metrics. This dataset also includes the rurality and economic need indicators modeled on the [USDA Community Connect](https://www.rd.usda.gov/programs-services/telecommunications-programs/community-connect-grants) grant criteria..
```

Comment:
```suggestion
Each point in this dataset was placed inside the boundary of a city, town, or CDP. Features in this dataset contain a wide variety of demographic data, including income, age, race, and ethnicity metrics. This dataset also includes the rurality and economic need indicators modeled on the [USDA Community Connect](https://www.rd.usda.gov/programs-services/telecommunications-programs/community-connect-grants) grant criteria..
```
I'm not exactly clear on how we created the point locations, but some are not in the "geographic center," so I just wanted to remove that part.